.github/workflows/coverage.yml

```diff
@@ -28,7 +28,13 @@ jobs:
           poetry install
 
       - name: Generate report
-        run: poetry run pytest tests/test_pytest.py --cov flexmock --cov-report xml
+        run: |
+          poetry run coverage run --parallel-mode --source flexmock -m unittest tests/test_flexmock.py
+          poetry run coverage run --parallel-mode --source flexmock tests/test_unittest.py
+          poetry run coverage run --parallel-mode --source flexmock tests/test_doctest.py
+          poetry run coverage run --parallel-mode --source flexmock -m pytest tests/test_pytest.py
```

Comment:
this feels a bit duplicate with tox, do you think we could run the coverage with it ?

Reply:
You are right. How about I create an issue to investigate this? I think tox runs every command in isolation so it would require some trickery to store the coverage reports for the `coverage combine` command.

Reply:
Yeah, it requires proper thinking! 😃

Reply:
I created an issue #117